company/careers.md

```diff
@@ -14,6 +14,7 @@ We're hiring! Check out our open positions:
 - [Software Engineer - Backend](../handbook/engineering/hiring/software-engineer-backend.md)
 - [Software Engineer - Distribution](../handbook/engineering/hiring/software-engineer-distribution.md)
 - [Software Engineer - Security](../handbook/engineering/hiring/software-engineer-security.md)
+- [Software Engineer - Code Insights - Frontend](../handbook/engineering/hiring/software-engineer-code-insights.md)
```

Comment:
Should we generally group them by team, or by role? (I'll also be adding a backend JD soon)

Reply:
Role makes more sense to me (an individual candidate who is a frontend candidate will be more interested in seeing all frontend roles rather than all roles on a single team)